client/src/common_css/site.scss

```diff
@@ -144,10 +135,6 @@ strong,
   font-weight: $mediumFontWeight;
 }
 
-.light-weight {
-  font-weight: $lightFontWeight;
-}
```

Comment:
I'd have said to leave this for consistency with our other font weight utility classes.